<ui version="4.0" >
 <class>EditMultipleWidget</class>
 <widget class="QWidget" name="EditMultipleWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>316</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>EditMultipleWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox6" >
     <property name="title" >
      <string>Select Objects To Edit</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="textLabel1" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>Filter:</string>
        </property>
        <property name="wordWrap" >
         <bool>false</bool>
        </property>
        <property name="buddy" >
         <cstring>_filterEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="QLineEdit" name="_filterEdit" />
      </item>
      <item row="3" column="0" colspan="3" >
       <widget class="QListWidget" name="_objectList" >
        <property name="selectionMode" >
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3" >
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <widget class="QPushButton" name="_selectNoneBut" >
          <property name="text" >
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="_selectAllBut" >
          <property name="text" >
           <string>Select &amp;all</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>_filterEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_selectAllBut</sender>
   <signal>clicked()</signal>
   <receiver>EditMultipleWidget</receiver>
   <slot>selectAllObjects()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_selectNoneBut</sender>
   <signal>clicked()</signal>
   <receiver>_objectList</receiver>
   <slot>clearSelection()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>136</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_filterEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>EditMultipleWidget</receiver>
   <slot>applyFilter(QString)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
